frontend/src/pages/BookGalleryPage/bookGalleryPage/BookGalleryPage.tsx

```diff
@@ -16,13 +17,59 @@ export default function BookGalleryPage({filteredBooks, setSearchInput}: BookGal
     const [statusFilter, setStatusFilter] = useState<string>("ALL")
 
 
+
+    const [showFilter, setShowFilter] = useState<boolean>(false);
+    const [selectedGenre, setSelectedGenre] = useState<string>("Select");
+    const [showFilterTag, setShowFilterTag] = useState<boolean>(false);
+
+    const handleClick = () => {
```

Comment:
handleClick is unnecessary, could just have onClick={()=>setShowFilter(!showFilter))? but if you want to keep it, I'd rather the function has a more meaningful name, like toggleFilter

Reply:
erledgit